base/deprecated.jl

```diff
@@ -1722,6 +1722,14 @@ export hex2num
 # PR #23373
 @deprecate diagm(A::BitMatrix) diagm(vec(A))
 
+# PR 23341
+@eval GMP @deprecate gmp_version() version() false
+@eval GMP @Base.deprecate_binding GMP_VERSION VERSION false
+@eval GMP @deprecate gmp_bits_per_limb() bits_per_limb() false
+@eval GMP @Base.deprecate_binding GMP_BITS_PER_LIMB BITS_PER_LIMB false
+@eval MPFR @deprecate get_version() version() false
+@eval LinAlg.LAPACK @deprecate laver() version() false
```

Comment:
true (anyone mind fixing this? I don't have time to do it)

Reply:
I think the best way to fix this is to leave `laver` as it was before this change and just add `version` since, generally, our LAPACK wrappers are pretty thin and don't add much Julia abstraction on top of LAPACK.